jbpm-bpmn2/src/main/java/org/jbpm/bpmn2/xml/TaskHandler.java

```diff
@@ -270,7 +270,7 @@ public Object end(final String uri, final String localName,
 				String uniqueId = (String) node.getMetaData().get("UniqueId");
 				forEachNode.setMetaData("UniqueId", uniqueId);
 				node.setMetaData("UniqueId", uniqueId + ":" + uniqueIdGen++);
-				node.setMetaData("hidden", true);
+//				node.setMetaData("hidden", true);
```

Comment:
Okay, would you mind removing the line then, instead of commenting it out? I think I would get confused by it I saw the commented-out line later.